github/repos_deployments_branch_policies.go

```diff
@@ -0,0 +1,119 @@
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+type DeploymentBranchPolicy struct {
+	Name   *string `json:"name,omitempty"`
+	ID     *int64  `json:"id,omitempty"`
+	NodeID *string `json:"node_id,omitempty"`
+}
+
+type DeploymentBranchPolicyResponse struct {
+	TotalCount     *int                      `json:"total_count,omitempty"`
+	BranchPolicies []*DeploymentBranchPolicy `json:"branch_policies,omitempty"`
+}
+
+type DeploymentBranchPolicyListOptions struct {
+	ListOptions
+}
+
+type DeploymentBranchPolicyRequest struct {
+	Name *string `json:"name,omitempty"`
+}
+
+// ListDeploymentBranchPolicies lists the deployment branch policies for an environment.
+//
+// GitHub API docs: https://docs.github.com/en/rest/deployments/branch-policies#list-deployment-branch-policies
+func (s *RepositoriesService) ListDeploymentBranchPolicies(ctx context.Context, owner, repo string, environment string) (*DeploymentBranchPolicyResponse, *Response, error) {
+	u := fmt.Sprintf("repos/%s/%s/environments/%s/deployment-branch-policies", owner, repo, environment)

```

Comment:
Please use `%v` for consistency and ease of searching through code for endpoints.